catalog/core/forms.py

```diff
@@ -52,6 +67,16 @@ def search(self):
         if self.cleaned_data['status']:
             criteria.update(status=self.cleaned_data['status'])
 
+        # Check to see if tags was selected.
+        if self.cleaned_data['tags']:
+            tag_objects = self.cleaned_data['tags']          
+            sqs = sqs.filter(reduce(operator.or_, (Q(tags=x) for x in tag_objects)))
```

Comment:
Is there any functional difference between this and `sqs.filter(tags__in=tag_objects)`? If not I'd prefer the shorter, simpler expression.